### PR TITLE
Improve makeblastdb support

### PR DESCRIPTION
**Pull request by eparejatobes**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -14,6 +14,8 @@ project/plugins/project/
 data/in/
 data/out/
 
+refs.fasta.*
+
```
Comment (laughedelic) [inline .gitignore]: This pattern misses `refs.fasta`

Comment (eparejatobes) [inline .gitignore]: it's intentional: I want the sequences in the repo, but not the BLAST db files.

**Comment by laughedelic** [comment]
@eparejatobes I'm going to make a release with #47, do you want to merge this? or it is not finished?

**Comment by eparejatobes** [comment]
@laughedelic I want to add proper tests here before merging.

**Comment by eparejatobes** [comment]
@laughedelic do you need this in mg7? I mean `-seqid_list`.

**Comment by laughedelic** [comment]
Ok. No problem. 
No it's not needed in MG7,  but it will be needed in projects, I guess,  and then it will be a bit problematic to combine different versions.  That's why I asked.

**Pull request merged**